services/ui-src/src/components/forms/AdminBannerForm.tsx

```diff
@@ -21,11 +21,11 @@ export const AdminBannerForm = ({ writeAdminBanner, ...props }: Props) => {
       link: formData["abf-link"],
       startDate: convertDatetimeStringToNumber(
         formData["abf-startDate"],
-        "abf-startDate"
+        formJson.fields.find((el) => el.id === "abf-startDate")!.props.timetype!
       ),
       endDate: convertDatetimeStringToNumber(
         formData["abf-endDate"],
-        "abf-endDate"
+        formJson.fields.find((el) => el.id === "abf-endDate")!.props.timetype!
```

Comment:
A changed method in `utils/time` now needs the "timeType" rather than the field name. More on this below.